src/signTxMint.c

```diff
@@ -240,9 +242,26 @@ static void signTxMint_handleToken_ui_runStep()
 		        this_fn
 		);
 	}
+	UI_STEP(HANDLE_TOKEN_STEP_DISPLAY_AMOUNT_WARNING) {
+		const int places = getTokenDecimalPlaces(
+		                           &subctx->stateData.tokenGroup,
+		                           subctx->stateData.token.assetNameBytes, subctx->stateData.token.assetNameSize
+		                   );
+		// this warning is only needed when we don't know decimal places
+		if (places != UNKNOWN_DECIMAL_PLACES) {
+			UI_STEP_JUMP(HANDLE_TOKEN_STEP_DISPLAY_AMOUNT);
+		}
+
+		ui_displayPaginatedText(
+		        "WARNING: unknown",
+		        "decimal places",
```

Comment:
I think this is a bit harsh, a simple **`INFO: unknown decimals`** is enough IMO. What do you think?

Reply:
I agree, I'll change this.